app/frontend/good_job/modules/metrics_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "stimulus"
+export default class extends Controller {
```

Comment:
I'd like this Stimulus controller to be written generically as like "AsyncValuesController". I'm imagining,

It would take a `url` value that corresponded to the value being fetched.

It would have `value` targets with a `key` attribute whose value corresponded to a key in the fetched json.

The resulting HTML would look something like this:

```html
<div data-controller="async-values" data-async-values-url="...">
  <div data-async-values-target="value" data-async-values-key="jobs_count"></div>
</div>
```